test/e2e/run_volume_test.go

```diff
@@ -665,6 +665,55 @@ VOLUME /test/`, ALPINE)
 		Expect(found).Should(BeTrue())
 	})
 
+	It("podman run with --mount and U flag", func() {
+		u, err := user.Current()
+		Expect(err).To(BeNil())
+		name := u.Username
+		if name == "root" {
+			name = "containers"
+		}
+
+		content, err := ioutil.ReadFile("/etc/subuid")
+		if err != nil {
+			Skip("cannot read /etc/subuid")
+		}
+
+		if !strings.Contains(string(content), name) {
+			Skip("cannot find mappings for the current user")
+		}
+
+		mountPath := filepath.Join(podmanTest.TempDir, "foo")
+		os.Mkdir(mountPath, 0755)
+
+		// bind mount
+		vol := "type=bind,src=" + mountPath + ",dst=" + dest + ",U=true"
+		session := podmanTest.Podman([]string{"run", "--rm", "--user", "888:888", "--mount", vol, ALPINE, "stat", "-c", "%u:%g", dest})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+		found, _ := session.GrepString("888:888")
+		Expect(found).Should(BeTrue())
+
+		// tmpfs mount
+		vol = "type=tmpfs," + "dst=" + dest + ",U=true"
+		session = podmanTest.Podman([]string{"run", "--rm", "--user", "888:888", "--mount", vol, ALPINE, "stat", "-c", "%u:%g", dest})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+		found, _ = session.GrepString("888:888")
+		Expect(found).Should(BeTrue())
+
+		// named volume mount
+		namedVolume := podmanTest.Podman([]string{"volume", "create", "foo"})
+		namedVolume.WaitWithDefaultTimeout()
+		Expect(namedVolume).Should(Exit(0))
+
+		vol = "type=volume,src=foo,dst=" + dest + ",U=true"
```

Comment:
Please also test `,U=false` and `,U=random`, see the other comment.

Reply:
done